src/serial/SerialArea.tsx

```diff
@@ -53,31 +53,31 @@ const SerialArea = ({
         position="relative"
         overflow="hidden"
       >
-        {!connected ? null : (
-          <Box
-            alignItems="stretch"
-            backgroundColor={backgroundColorTerm}
-            height="100%"
-          >
-            <SerialBar
-              height={12}
-              compact={compact}
-              onSizeChange={onSizeChange}
-              showSyncStatus={showSyncStatus}
-              expandDirection={expandDirection}
-              hideExpandTextOnTraceback={hideExpandTextOnTraceback}
-              showHintsAndTips={showHintsAndTips}
-            />
-            <XTerm
-              visibility={compact ? "hidden" : undefined}
-              height={`calc(100% - ${SerialArea.compactSize}px)`}
-              ml={1}
-              mr={1}
-              fontSizePt={terminalFontSizePt}
-              tabOutRef={tabOutRef}
-            />
-          </Box>
-        )}
+        <Box
+          // Need to render this when not connected as we need it to maintain scrollback across disconnect/reconnect in some cases.
+          display={connected ? undefined : "none"}
```

Comment:
Until this change reset wasn't really what was resetting the terminal, as we were also discarding it entirely.

Need to review for unexpected consequences here.